Description: Prevent non-evdev devices from firing the eventcomm AutoDevProbe callback.
Bug-Ubuntu: https://launchpad.net/bugs/624985

Index: xserver-xorg-input-synaptics-1.8.2/conf/50-synaptics.conf
===================================================================
--- xserver-xorg-input-synaptics-1.8.2.orig/conf/50-synaptics.conf
+++ xserver-xorg-input-synaptics-1.8.2/conf/50-synaptics.conf
@@ -14,7 +14,7 @@ Section "InputClass"
 # This option is recommend on all Linux systems using evdev, but cannot be
 # enabled by default. See the following link for details:
 # http://who-t.blogspot.com/2010/11/how-to-ignore-configuration-errors.html
-#       MatchDevicePath "/dev/input/event*"
+      MatchDevicePath "/dev/input/event*"
 EndSection
 
 Section "InputClass"
